extensions/amp-story/0.1/amp-story.js

```diff
@@ -314,16 +314,23 @@ export class AmpStory extends AMP.BaseElement {
   buildButtons_() {
     const doc = this.element.ownerDocument;
     const nextButton = doc.createElement('button');
+    const nextButtonContainer = doc.createElement('div');
```

Comment:
nit: this DOM tree is getting too complicated for a series of `createElement`'s. Consider using utilities found in `./simple-template` (see `system-layer.js` for an example).

Reply:
Done 👍